moabb/evaluations/evaluations.py

```diff
@@ -59,7 +59,7 @@ def __init__(
                     "Number of elements in n_perms must be equal "
                     "to number of elements in data_size['value']"
                 )
-            elif np.all(np.diff(n_perms) <= 0):
+            elif not np.all(np.diff(n_perms) <= 0):
```

Comment:
There is still a problem: it works only if all n_perms values are identical 
```
n_perms = array([10, 10, 10, 10])
print (np.all(np.diff(n_perms) <= 0))  # this is True
n_perms = np.array([5, 10, 10, 15]) 
print (np.all(np.diff(n_perms) <= 0))  # this is False while monotonically increasing
n_perms = np.array([10, 5, 1, 15])
print (np.all(np.diff(n_perms) <= 0))  # this is False and it should be

Reply:
I think you should use:
`if not np.all(np.diff(n_perms) >= 0):`

Reply:
Hm, I always used it such that the data sizes are increasing and the permutations are decreasing, i.e.:
```
data_size=array([0.1, 0.5, 1])
n_perms=array([20,10,3])
```
Then I can match the index of data_size to the index of n_perms.

Maybe I should enforce that data size is strictly monotonically increasing such that this is clear?

Reply:
It makes sense, you are right, we need to make sure that the `n_perms` is decreasing and `data_size` is increasing. We should add some test for that in `moabb/tests/evaluations.py`, to be sure that we don't introduce some regression in the code.

Reply:
I added some unit tests for the learning curve evaluation.